packages/main/src/SliderBase.js

```diff
@@ -0,0 +1,648 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Float from "@ui5/webcomponents-base/dist/types/Float.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import { getTheme } from "@ui5/webcomponents-base/dist/config/Theme.js";
+
+// Styles
+import styles from "./generated/themes/SliderBase.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	properties: /** @lends sap.ui.webcomponents.main.SliderBase.prototype */  {
+		/**
+		 * Defines the minimum value of the slider
+		 *
+		 * @type {Float}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		min: {
+			type: Float,
+			defaultValue: 0,
+		},
+		/**
+		 * Defines the maximum value of the slider
+		 *
+		 * @type {Float}
+		 * @defaultvalue 100
+		 * @public
+		 */
+		max: {
+			type: Float,
+			defaultValue: 100,
+		},
+		/**
+		 * Defines the size of the slider's selection intervals. (e.g. min = 0, max = 10, step = 5 would result in possible selection of the values 0, 5, 10).
+		 * If set to 0 the slider handle movement is disabled. When negative number or value other than a number, the component fallbacks to its default value.
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 1
+		 * @public
+		 */
+		step: {
+			type: Float,
+			defaultValue: 1,
+		},
+		/**
+		 * Displays a label with a value on every N-th step. The step and tickmarks properties must be enabled.
```

Comment:
1. "than" should be "then" I guess
2. Use Note for this part "If set to 0.." :
Use note here as well
```html
* Displays a label with a value on every N-th step.
* <br>
* Example: if the <code>step</code> value is set to 2 and the <code>label interval</code> is also specified to 2,  
then every second tickmark will be labelled, which means every 4th value number.
* <br><br>
* <b>Note:</b> The step and tickmarks ..
```